plan/templates/plan/index.html

```diff
@@ -0,0 +1,11 @@
+{% extends "dashboard/action.html" %}{% load bootstrap3 %}
+
+{% block title%}Plan{% endblock %}
+
+{% block content_dashboard_view %}
+    <link rel='stylesheet' href='//cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.3.1/fullcalendar.min.css' />
```

Comment:
This is a work in progress addition to test the calendar. Actually the whole file should not be in here